Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -1031,7 +1031,7 @@ public struct RenderNodeTranslator: SemanticVisitor {
                 .filter({ !($0.unconditionallyUnavailable == true) })
                 .sorted(by: AvailabilityRenderOrder.compare)
         } ?? .init(defaultValue:
-            defaultAvailability(for: bundle, moduleName: moduleName, currentPlatforms: context.externalMetadata.currentPlatforms)?
+            defaultAvailability(for: bundle, moduleName: moduleName.symbolName, currentPlatforms: context.externalMetadata.currentPlatforms)?
```

Comment:
Is there value in storing around the `symbolName` in the `moduleNameCache` or should/can we consider just pulling this value from the `Symbol` semantic the way this logic used to work before #92?

Reply:
If we need to display a module's name somewhere else in the future I feel that there is a little bit of value in having one code path to get "the name of a module" that returns one value where the module's different names can be accessed.

That way, the person who implements that in the future doesn't need to know that a module can names for different purposes and doesn't need to think about which name they should get before accessing that name on the return value.

Reply:
That makes a lot of sense to me. Thank you!